tools/cockpit.spec

```diff
@@ -219,6 +233,10 @@ BuildRequires:  python3-tox-current-env
 %if 0%{?is_smo}
 %patch101 -p1
 %patch102 -p1
+%else
```

Comment:
no need %else here, I think

Reply:
Here I think we want to check just for SLES 15 SPX codebase. That will cover SMO too.

Reply:
I mean insstead of

```
%else
%if 0%{?sle_version} >= 150400 && 0%{?sle_version} <= 150700
%patch103 -p1
%endif
%endif

```

just have

```
%if 0%{?sle_version} >= 150400 && 0%{?sle_version} <= 150700
%patch103 -p1
%endif
```

Or even to simplify this to

`%if 0%{?suse_version} == 1500` because this problem also affects smo, they just don't know it yet ;)